src/code/z_actor.c

```diff
@@ -4041,6 +4041,8 @@ void func_8003424C(PlayState* play, Vec3f* arg1) {
 }
 
 void Actor_SetColorFilter(Actor* actor, s16 colorFlag, s16 colorIntensityMax, s16 bufFlag, s16 duration) {
+    //! @bug This first comparison is always false as COLORFILTER_COLORFLAG_GRAY = 0x8000, but colorFlag is an s16
+    //! so cannot be equal to 0x8000 as sign extension results in comparing 0xFFFF8000 == 0x00008000 which is false.
```

Comment:
I found this explanation confusing, it seems to imply that `colorFlag` guaranteed to be -0x8000 whenever this function is called. Maybe it would be more concise to say that +0x8000 is out of range of an s16?